web/src/app/forms/FormField.js

```diff
@@ -226,4 +226,6 @@ FormField.propTypes = {
   type: p.string,
   select: p.bool,
   timeZone: p.string,
+
+  userID: p.string,
```

Comment:
not sure if we should include this here, `<FormField />` can be used pretty generally by a number of components, for example a timezone select dropdown that wouldn't have a userID on hand. which component was complaining that you needed to add this?

Reply:
This is the error I get when I remove `userID: p.string,` from the FormField props:
<img width="966" alt="Screen Shot 2022-05-11 at 3 24 18 PM" src="https://user-images.githubusercontent.com/52386267/167941415-0b19e6c6-52fe-4bca-9ab3-8343c7e4afe4.png">

Reply:
Gotcha, looks like it's being passed through to `UserContactMethodSelect` and isn't actually being used here (the line where `...otherFieldProps` is instantiated).

I don't think we can avoid needing to define it so it's probably fine to make it optional

```
userID?: p.string,
```